packages/devextreme/testing/tests/DevExpress.ui.widgets/resizeHandle.markup.tests.js

```diff
@@ -168,6 +168,32 @@ QUnit.module('ResizeHandle markup', moduleConfig, () => {
                 assert.strictEqual(this.$element.css(dimension), '8px');
             });
 
+            [
+                { separatorSize: '50vh' }, { separatorSize: '20spx' }, { separatorSize: 'd10' }, { separatorSize: 'NaN' },
```

Comment:
add value with spaces ' 100 ', ' 100px ',

Reply:
why do you think ' 100' is a bad input?